configure

```diff
@@ -1027,6 +1031,76 @@ else
     echo "Checking for __builtin_ctzll ... No." | tee -a configure.log
 fi
 
+check_avx512vnni_intrinsics() {
+    # Check whether compiler supports AVX512-VNNI intrinsics
+    cat > $test.c << EOF
+"#include <stdio.h>
```

Comment:
Do you need to include stdio.h?

Reply:
Ugh, no. Lol I don't know how that header keeps sneaking in. I'll remove it.